packages/storybook/stories/va-radio.stories.jsx

```diff
@@ -80,9 +84,35 @@ const IdUsageTemplate = ({
   );
 };
 
+const OptionDescriptionTemplate = ({
+  'enable-analytics': enableAnalytics,
+  error,
+  label,
+  description,
+  required,
+}) => {
+  return (
+    <va-radio
+      enable-analytics={enableAnalytics}
+      error={error}
+      label={label}
+      required={required}
+    >
+      <va-radio-option label="Option one" name="example" value="1">
+        <div>Option one description</div>
+      </va-radio-option>
+      <va-radio-option label="Option two" name="example" value="2">
+        <div>Option two longer description</div>
+      </va-radio-option>
+    </va-radio>
+  );
+};
+
 const defaultArgs = {
   'enable-analytics': false,
   'label': 'This is a label',
+  'hint': '',
```

Comment:
The Storybook page is displaying `hint` in a strange place above the component properties. 

![Screen Shot 2022-09-21 at 9 57 00 AM](https://user-images.githubusercontent.com/872479/191540088-6ff7b801-22c9-43bf-a8a8-fdb43b7a5d48.png)

Is this because you're passing it as a default argument that isn't mapping to any actual property in the component? Or is Storybook picking up the named slot?

Reply:
I'm not sure how to fix this... the same thing happens with the [accordion headline entry](https://design.va.gov/storybook/?path=/docs/components-va-accordion--default)

Reply:
@bkjohnson I guess this is getting picked up because of the named slot. Is that expected? Are we ok with it displaying like that?

Reply:
[`va-select` has the same pattern](https://design.va.gov/storybook/?path=/docs/components-va-select--default)... the options added to the default args are showing up

Reply:
Yeah, Robin is right about the `defaultArgs`. I wouldn't say that it's an ideal situation.